server/index.js

```diff
@@ -24,8 +24,8 @@ app.use(
   session({
     secret: process.env.EXPRESS_SESSION_KEY,
     unset: 'destroy',
-    saveUninitialized: true,
-    resave: true
+    saveUninitialized: false,
+    resave: false
```

Comment:
just curious, what these changes are doing>>

Reply:
resave: true - forces the session to be saved back to the session store, even if the session was never modified...
saveUninitialized: true - forces a session that is "uninitialized" to be saved to the store.
Source: https://www.npmjs.com/package/express-session